contrib/opencensus-ext-azure/opencensus/ext/azure/common/protocol.py

```diff
@@ -65,6 +65,25 @@ def __init__(self, *args, **kwargs):
         self.baseType = self.baseType
 
 
+class DataPoint(BaseObject):
+    _default = BaseObject(
+        ns='',
+        name='',
+        kind=None,
+        value=0.0,
+        count=None,
+        min=None,
+        max=None,
+        stdDev=None,
+    )
+
+    def __init__(self, *args, **kwargs):
+        super(DataPoint, self).__init__(*args, **kwargs)
+        self.ns = self.ns
```

Comment:
`ns` is not required here?

Reply:
Can you explain a little bit about why it is not required?

Reply:
According to https://github.com/microsoft/ApplicationInsights-Home/blob/master/EndpointSpecs/Schemas/Bond/DataPoint.bond, ns is not required.

Reply:
Yes. After figuring out the behavior of the injection service, I removed ns as a required field.